cwltool/job.py

```diff
@@ -543,7 +543,7 @@ def create_file_and_add_volume(self,
             tmp_dir, tmp_prefix = os.path.split(tmpdir_prefix)
             new_file = os.path.join(
                 tempfile.mkdtemp(prefix=tmp_prefix, dir=tmp_dir),
-                os.path.basename(volume.resolved))
+                os.path.basename(volume.target))
```

Comment:
Another bug fix? Again, would like to see a unit test & conformance test that triggers this bug.

Reply:
This is definitely a bug fix, but I have to go back and look at exactly what was happening.